netbox/data_source_netbox_devices_test.go

```diff
@@ -204,6 +206,7 @@ resource "netbox_custom_field" "test" {
 resource "netbox_device" "test" {
   name = "%[2]s"
   comments = "thisisacomment"
+	description = "thisisadescription"
```

Comment:
>:[

Reply:
grrr.... my editor is being....annoying! 😞